### ISSUE TITLE
flash space not enough for esp32c3

### ISSUE THREAD

**Issue by dennis4lora** [lora-gateway/esxp1302#14]
When building for `esp32c3` using **PlatformIO**, it failed with below error:

```
...
Linking .pio/build/esp32c3/firmware.elf
Retrieving maximum program size .pio/build/esp32c3/firmware.elf
Checking size .pio/build/esp32c3/firmware.elf
Advanced Memory Usage is available via "PlatformIO Home > Project Inspect"
Error: The program size (1050666 bytes) is greater than maximum allowed (1048576 bytes)
*** [checkprogsize] Explicit exit, status 1
RAM:   [===       ]  33.2% (used 108836 bytes from 327680 bytes)
Flash: [==========]  100.2% (used 1050666 bytes from 1048576 bytes)
```

So it's time to take some actions to optimize the flash usage now...

**Comment by danielkucera** [comment]
No, you were right. Fix in #15 .

**Comment by dennis4lora** [comment]
Verified and it's fixed. Thanks.
